src/utils/helpers.js

```diff
@@ -27,6 +27,10 @@ export function getCurrentMonth() {
   return getUKFormattedDate(Date.now(), { year: 'numeric', month: 'long' });
 }
 
+export function getExpenseMonth(expense) {
```

Comment:
interesting. my object orientation heat now jumps in:

what if expense were a well known object/class. the you could ask it to get the month from it and you could ask it to update itself in the database or remove itself from the database. this is very common in model-view-controller architectures. where you have ORMs. here an example:

```js

expenses = Expense.getAllExpenses() // static method in the expense class

expense = expenses[0]
expense.title // -> "Fish"
expense.amount // use the object as usual for displaying stuff
expense.update(title: "Cheese") // but also for more sophisticated stuff. e.g. here it sends a request to the database
expense.isValid()
expense.delete()
```